lib/node/runner/api/api.go

```diff
@@ -61,45 +61,56 @@ func (api NetworkHandlerAPI) HandlerURLPattern(pattern string) string {
 }
 
 func TriggerEvent(st *storage.LevelDBBackend, transactions []*transaction.Transaction) {
+	var (
+		t     = obs.ResourceObserver.Trigger
+		cond  = obs.NewCondition
+		event = obs.Event
+	)
+
 	accountMap := make(map[string]struct{})
+
 	for _, tx := range transactions {
 		source := tx.Source()
 		accountMap[source] = struct{}{}
 		txHash := tx.H.Hash
+		bt, err := block.GetBlockTransaction(st, tx.H.Hash)
+		if err != nil {
```

Comment:
At the streaming mode of subscribe api,
If there is an error when getting the block data, just ignore it.